tests/snmpsim/ciscosb_cbs250-24p-4x.snmprec

```diff
@@ -2058,15 +2058,15 @@
 1.3.6.1.2.1.31.1.1.1.18.3|4|
 1.3.6.1.2.1.31.1.1.1.18.4|4|
 1.3.6.1.2.1.31.1.1.1.18.5|4|
-1.3.6.1.2.1.31.1.1.1.18.6|4x|447275636B657220536368756C62C3BC0a373220364620
+1.3.6.1.2.1.31.1.1.1.18.6|4x|Drucker
```

Comment:
Hi @Fehler12 
Did you manually change this or did you use the script ? I would say that this line expects HEX data and not string. So "Drucker" should be `447275636B6572`.

Reply:
yes and no: For values with only normal characters the SNMPREC gives a cleartext value. For values with special characters (umlauts in my case) it seems that they get decoded in HEX-format. To make it easier for me I copied a value of another network port to this port because it seems to me that your test systems and my system handle umlauts differently and therefore the checks fail.